isotovideo

```diff
@@ -85,14 +85,8 @@ no autodie 'kill';
 use constant {
     EXIT_STATUS_OK => 0,
     EXIT_STATUS_ERR => 1,
-    EXIT_STATUS_ERR_NO_TESTS => 100,
-    EXIT_STATUS_ERR_FROM_TEST_RESULTS => 101,
 };
 
-# Avoid "Subroutine JSON::PP::Boolean::(0+ redefined" warnings
-# Details: https://progress.opensuse.org/issues/90371
-use JSON::PP;
-
```

Comment:
These 3 lines need to stay here. It's a bug introduced by Cpanel::JSON::XS (it's creating the same functions as JSON::PP) and fixed in newer versions of JSON::PP, however we are still on a perl version where we don't have the newer JSON::PP.
In the module, just do `use JSON::PP;`